src/test/db_benchmark/common.ml

```diff
@@ -0,0 +1,117 @@
+open Core
+
+(* Configuration constants - configurable via environment variables *)
+let keys_per_block =
+  Sys.getenv "KEYS_PER_BLOCK" |> Option.value_map ~default:125 ~f:int_of_string
+
+let value_size =
+  Sys.getenv "VALUE_SIZE"
+  |> Option.value_map ~default:(128 * 1024) ~f:int_of_string
+(* 128 KB *)
+
+let warmup_blocks =
+  Sys.getenv "WARMUP_BLOCKS" |> Option.value_map ~default:800 ~f:int_of_string
+
+(* Fixed seed for reproducibility *)
+let random_seed = 42
```

Comment:
Would be nice to make this random if not user provided. And the test prints out the seed it's using every time.